ERROR - 2023-09-21 14:38:16 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\laragon\www\lab1\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\lab1\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\laragon\www\lab1\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\lab1\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\lab1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\lab1\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\lab1\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\lab1\app\Controllers\ProductController.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\lab1\system\CodeIgniter.php(919): App\Controllers\ProductController->index()
#8 C:\laragon\www\lab1\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 C:\laragon\www\lab1\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\lab1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\lab1\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\laragon\www\lab1\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\lab1\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\lab1\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\lab1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\lab1\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\lab1\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\lab1\app\Controllers\ProductController.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\lab1\system\CodeIgniter.php(919): App\Controllers\ProductController->index()
#7 C:\laragon\www\lab1\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 C:\laragon\www\lab1\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\lab1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\lab1\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2023-09-21 14:38:16 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `table_laptop`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ProductController.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->index()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\lab1\\public\\index.php')
CRITICAL - 2023-09-21 14:39:18 --> Invalid file: "products/index.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products/index.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('products/index', [], true)
 3 APPPATH\Controllers\ProductController.php(14): view('products/index', [...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\lab1\\public\\index.php')
CRITICAL - 2023-09-21 14:39:57 --> Undefined variable $categorLModel
in APPPATH\Controllers\ProductController.php on line 19.
 1 APPPATH\Controllers\ProductController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $categorLModel', 'C:\\laragon\\www\\lab1\\app\\Controllers\\ProductController.php', 19)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->create()
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\lab1\\public\\index.php')
